### PR TITLE
Improve CMakeLists.txt: support "make install"

### PR DESCRIPTION
**Pull request by SunBeau**
Set "-std=gnu99" to fix compilation errors

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -93,3 +95,8 @@ TARGET_LINK_LIBRARIES(qlibc PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 TARGET_LINK_LIBRARIES(qlibcext-static PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 TARGET_LINK_LIBRARIES(qlibcext PUBLIC qlibc)
 
+INSTALL(DIRECTORY ${qlibc_SOURCE_DIR}/include/qlibc DESTINATION include)
+INSTALL(TARGETS   qlibc           LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibc-static    LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibcext        LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibcext-static LIBRARY           DESTINATION lib)
```
Comment (wolkykim) [inline CMakeLists.txt:102]: Hi,  what do these lines do?

**Comment by wolkykim** [comment]
sorry, I didn't notice this PR. It looks good to me. Just left a question.

File: CMakeLists.txt (hunk 1 above)
Comment (SunBeau) [reply CMakeLists.txt:102]: Use configure  to compile
```shell
./configure --prefix=/tmp/libqlibc
make
make install # This step will install header and library files to /tmp/libqlibc

```

Use CMakeLists.txt to compile
```shell
mkdir build
cd build/
cmake -DCMAKE_INSTALL_PREFIX=/tmp/libqlibc ..
make
make install # This step will report an error
```

The function of these lines is that when using CMakeLists.txt to compile, "make install" can be executed normally.

Comment (wolkykim) [reply CMakeLists.txt:102]: Thank you for the detailed explanations. Looks great!

**Review by wolkykim — APPROVED** [review]
Thank you for your contribution and the great PR.

**Pull request merged**